src/browser/setupWorker/setupWorker.ts

```diff
@@ -177,6 +178,14 @@ export class SetupWorkerApi
       options,
     ) as SetupWorkerInternalContext['startOptions']
 
+    invariant(
+      !this.context.startOptions.dangerouslyRunInProduction && isProduction(),
+      devUtils.formatMessage(
+        'The flag dangerouslyRunInProduction is false but you are in a production environment',
+      ),
+      'https://github.com/mswjs/msw/issues/1703',
```

Comment:
I think we should rather feature a link to the docs explaining this flag and what it does. I will document it.

Reply:
@kettanaito Awesome, that would be more appropriate.